templates/CRM/Contribute/Form/Contribution/MembershipBlock.tpl

```diff
@@ -134,7 +131,7 @@
   </div>
 
 {/if}{* membership block end here *}
```

Comment:
Could we put the js right before this if - this if is effectively 'if this is the Main page'- so it would always be present  - I have a feeling   `{elseif $lineItem and $priceSetID AND !$is_quick_config}` is never true but it seems to be in there

Reply:
@eileenmcnaughton that `{/if}` is after a couple of `{elseif}`s, so not so much, no, as that will leave it outside of the makeContribution context and it won't do anything useful there, rendering the move pointless.  This is one of the changes in the commit that introduced this regression.

Right *after* the `{if}` might make sense; **however** it's worth noting that there's no autorenewal checkbox on either the review or thank you pages, meaning it's pointless on those pages anyway.

Reply:
@agileware-fj yep - if should only be in the context = makeContribution - & always there when it is I think?